### PR TITLE
test(app): adding connect to IP slideout e2e tests

### PR DESCRIPTION
**Pull request by reachojhaneha**
Wrtiting the e2e testcase for connect to IP address

closes: [e2e ticket](https://app.zenhub.com/workspaces/app--ui-5f6248dba845b000139aa767/issues/opentrons/opentrons/9838)

### CODE REVIEW

File: app-testing/tests/app_settings_v5dot1_test.py [hunk 1]
```diff
@@ -78,6 +78,20 @@ def test_app_settings_v5dot1(
...
+        assert (
+            app_settings.get_connect_to_robot_via_IP_address_slideout_header().is_displayed()
+        )
+        assert (
+            app_settings.get_link_learn_more_about_connecting_a_robot_manually().is_displayed()
+        )
+        assert app_settings.get_textbox_to_enter_the_ip().is_displayed()
+        app_settings.click_add_ip_or_hostname()
+        assert app_settings.get_try_again_link().is_displayed()
+        app_settings.enter_hostname("AKNA")
```
Comment (koji) [inline app-testing/tests/app_settings_v5dot1_test.py]: This should be string[] instead of `string`
```python
app_settings.enter_hostname(["AKNA"])
```

**Comment by koji** [comment]
@reachojhaneha Sarah's latest PR has been merged into `edge`, so slideout would be okay.
I checked it manually.

**Review by koji — CHANGES_REQUESTED** [review]
`app_settings_v5dot1_test.py` requires a small change (L90).

```
app_settings.enter_hostname(["AKNA"])
```

**Comment by reachojhaneha** [comment]
@koji your comment has been addressed

**Review by koji — APPROVED** [review]
Looks good to me!

**Pull request merged**